.aicoe-ci.yaml

```diff
@@ -2,7 +2,7 @@ check:
   - thoth-precommit
   - thoth-build
 build:
-  base-image: "quay.io/thoth-station/s2i-thoth-f32-py38-cloc:latest"
+  base-image: quay.io/thoth-station/s2i-thoth-ubi8-py38:latest
```

Comment:
@fridex @KPostOffice Should I leave it as it is without this change?

Reply:
Do we have any specific reason why f32 should be used instead of ubi8?

Reply:
The cloc package cant be installed in ubi8 ,so we had to move along with f32.

Reply:
@harshad16 Should I put it back?

Reply:
i think we should just close this pr, as the app is already using py38.

Reply:
nice, thanks

Reply:
ah, that thoth.yaml wasnt update with this pr, that would get updated.
can you just put the image back, so that we can approve this.

Reply:
got it, thanks